install.sh

```diff
@@ -98,56 +98,33 @@ function curl () {
   return 1
 }
 
-case "${ARCH}" in
-"i686"|"x86_64"|"armv7l"|"aarch64")
-  LIB_SUFFIX=
-  [ "${ARCH}" == "x86_64" ] && LIB_SUFFIX='64'
-  ;;
-*)
-  echo_error "Your device is not supported by Chromebrew yet :/"
-  exit 1;;
-esac
-
 echo_info "\n\nDoing initial setup for install in ${CREW_PREFIX}."
 echo_info "This may take a while if there are preexisting files in ${CREW_PREFIX}...\n"
 
+# Delete ${CREW_PREFIX}/{var,local} symlinks which exist on some Chromium OS distros.
+[ -L ${CREW_PREFIX}/var ] && sudo rm -f "${CREW_PREFIX}/var"
+[ -L ${CREW_PREFIX}/local ] && sudo rm -f "${CREW_PREFIX}/local"
+
 # This will allow things to work without sudo.
-crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var"
+crew_folders="bin cache doc docbook include lib/crew/packages lib$LIB_SUFFIX libexec man sbin share var etc/crew/meta tmp/crew/dest"
 for folder in $crew_folders
 do
-  if [ -d "${CREW_PREFIX}"/"${folder}" ]; then
-    echo_intra "Resetting ownership of ${CREW_PREFIX}/${folder}"
-    sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"${folder}"
+  # This will create the directories if they do not exist
+  if [ ! -d "${CREW_PREFIX}"/"${folder}" ]; then
+    mkdir -p "${CREW_PREFIX}"/"${folder}"
+    echo_intra "Creating "${CREW_PREFIX}"/"${folder}""
   fi
```

Comment:
How about instead of the if statement?
```
rm -rf "${CREW_PREFIX}"/"${folder}"
mkdir -p "${CREW_PREFIX}"/"${folder}"
echo_intra "Creating "${CREW_PREFIX}"/"${folder}"
```
This makes sure the folder is clean before installing anything.